modules/appeals_api/app/serializers/appeals_api/notice_of_disagreement_serializer.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AppealsApi::NoticeOfDisagreementSerializer
+  include FastJsonapi::ObjectSerializer
+  set_key_transform :camel_lower
+  attributes :updated_at, :created_at, :form_data
```

Comment:
Are we missing an attribute for status?

Reply:
Status wasn't included in the migration. I checked with Ed and Steven and they said to add it separately in another ticket/PR. Let me know if I should do this sooner than later.